src/olympia/lib/settings_base.py

```diff
@@ -674,7 +680,6 @@ def get_db_config(environ_var, atomic_requests=True):
         'preload': (
```

Comment:
Could you rename this bundle to something like `jquery_base` or something and update references to it in `base.html` and `devhub/index.html` ? I find it confusing that it's still called `preload` but doesn't include the new `preload.js` file...

Reply:
Yes. FWIW, the goal is to eventually include all of that into a single preload bundle but that doesn't work when we are not bundling everything with vite so would happen later.